src/modules/user/mock/user.ts

```diff
@@ -0,0 +1,22 @@
+export type User = {
+  id: string;
+  name: string;
+  image: string;
```

Comment:
May be use `image?: string;`

Reply:
I think image will not be null it will be "" ?